web/src/helpers/translation.js

```diff
@@ -39,4 +39,21 @@ export function getFullZoneName(zoneCode) {
   return `${zoneName} (${countryName})`;
 }
 
+export function getShortZoneName(zoneCode, limit = 40) {
+  const zoneName = translate(`zoneShortName.${zoneCode}.zoneName`);
+  if (!zoneName) {
+    return zoneCode;
+  }
+  const countryName = translate(`zoneShortName.${zoneCode}.countryName`);
+  if (!countryName) {
+    return zoneName;
+  }
+
+  if (zoneName.length > limit) {
+    return `${zoneName.substring(0, limit)}... (${countryName})`;
+  }
+
+  return `${zoneName} (${countryName})`;
```

Comment:
Wouldn’t we just do this with css and a max width?

Reply:
Nope, the reason is that I want to shorten the zoneName, but not the countryName - so fx `Somethi... (USA)` instead of `Something (U...`.
 I guess we could split the HTML into two elements and do some work there, but I think this is much simpler for now :)